src/components/GenreMenuBar/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react'
+
+import './styles/styles.scss'
+
+const labels = ['ALL', 'DOCUMENTARY', 'COMEDY', 'HORROR', 'CRIME']
+
+const GenreMenuBar = () => {
+  const [genre, setGenre] = useState(labels[0])
+
+  return (<nav className={'genre-menu-bar genre-menu-bar__black'}>
```

Comment:
It is not necessary to use brackets for className here, if it is string value just pass it `className="genre-menu-bar genre-menu-bar__black"`